examples/paho-mqtt/Makefile

```diff
@@ -0,0 +1,36 @@
+APPLICATION = paho-mqtt-example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+USEPKG += paho-mqtt
+
+# paho-mqtt depends on TCP support, choose the stack you want
+USEMODULE += lwip_ipv6_autoconfig
+USEMODULE += lwip lwip_sock_ip
+USEMODULE += lwip_netdev
+USEMODULE += lwip_udp lwip_sock_udp
+USEMODULE += lwip_tcp lwip_sock_tcp
+USEMODULE += lwip_sock_async
+USEMODULE += sock_async_event
+USEMODULE += tsrb
```

Comment:
`tsrb` on the other hand should be a dependency of `paho-mqtt` (assuming it requires `malloc`)